hack/push-chart.sh

```
@@ -44,6 +45,20 @@ then
+    exit 1
+  fi
+  ${YQ} -i \
+    '.latencyPredictor.trainingServer.image.registry=strenv(IMAGE_REGISTRY) |
+     .latencyPredictor.trainingServer.image.repository="llm-d-latency-predictor-training-server" |
+     .latencyPredictor.trainingServer.image.tag=strenv(LATENCY_PREDICTOR_TAG) |
+     .latencyPredictor.trainingServer.image.pullPolicy="IfNotPresent" |
+     .latencyPredictor.predictionServers.image.registry=strenv(IMAGE_REGISTRY) |
+     .latencyPredictor.predictionServers.image.repository="llm-d-latency-predictor-prediction-server" |
+     .latencyPredictor.predictionServers.image.tag=strenv(LATENCY_PREDICTOR_TAG) |
+     .latencyPredictor.predictionServers.image.pullPolicy="IfNotPresent"' \
+    config/charts/${CHART}/values.yaml
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)